File: TourismAgency-frontend/src/app/util/oter/erole.ts
export class ERole {
    public static CUSTOMER:boolean=false;
    public static EMPLOYEE:boolean=false;
    public static OWNER:boolean=false; 
    public static ADMINISTRATOR:boolean=false;

    public static ERoleList(): boolean[]{
        return [ERole.CUSTOMER, ERole.EMPLOYEE, ERole.OWNER, ERole.ADMINISTRATOR];
    }
}

File: TourismAgency-backend/tourism/src/main/java/com/tourism/util/other/ERole.java
package com.tourism.util.other;

import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

public enum ERole {
    CUSTOMER, EMPLOYEE, OWNER, ADMINISTRATOR;

    public static List<String> getListERole(){
        return Arrays.stream(ERole.values()).map(Enum::name).collect(Collectors.toList());
    }
}
